exercises/template-driven-forms/README.md

```diff
@@ -2,4 +2,75 @@
 
 Create a template driven form with the following behavior
 
-![](./preview.gif)
\ No newline at end of file
+![](./preview.gif)
+
+## Step 1
+
+Using the `angular-cli` create a new project called `template-driven-forms` inside of the `training` folder.
+
+## Step 2
+
+Create a Typescript `interface` called `Person` that captures the data model of the form in a file called `person.model.ts`
+
+## Step 3
+
+Modify the `app.component.ts` to import the `Person` created in Step 2 and declare a property of that type on your `AppComponent`.  Also import the `NgForm` directive.
```

Comment:
I meant import { NgForm } from '@angular/forms';
https://angular.io/docs/ts/latest/api/forms/index/NgForm-directive.html

do you think that is too much information?  Should I remove or change?